pubnet/core/etc/stellar-core.cfg

```diff
@@ -7,6 +7,8 @@ DATABASE="postgresql://dbname=core host=localhost user=stellar password=__PGPASS
 NETWORK_PASSPHRASE="Public Global Stellar Network ; September 2015"
 CATCHUP_RECENT=100
 
+EXPERIMENTAL_BUCKETLIST_DB=true
```

Comment:
Do we need to set `EXPERIMENTAL_BUCKETLIST_DB_INDEX_PAGE_SIZE_EXPONENT=12`? That's what we set for the rpc which already uses the bucketlistdb in the quickstart image.

Reply:
This one is set to 14 in the [example config](https://github.com/SirTyson/stellar-core/blob/master/docs/stellar-core_example.cfg) in core repo. Also these two can be set?
```
BUCKETLIST_DB_INDEX_CUTOFF = 20
BUCKETLIST_DB_PERSIST_INDEX = true
```

Reply:
No, the default settings for these configs is appropriate for quick start so we don't need to set them. However we should change this flag to `DEPRECATED_SQL_LEDGER_STATE = false` instead of the experimental flag, as rc2 supports the new flag and we will eventually get rid of the old experimental flag.

Reply:
We can only use the new flag if cfg files if all versions of stellar-core across the three image types support that new flag.

Reply:
@SirTyson Should we remove the existing `EXPERIMENTAL_BUCKETLIST_DB_INDEX_PAGE_SIZE_EXPONENT=12` configs from the other cfgs in quickstart?

Reply:
I think we can do the cleanup separately (maybe combined with the BL DB flag switch).